.github/ISSUE_TEMPLATE/page-error.md

```diff
@@ -0,0 +1,39 @@
+---
+name: Page Error
+about: Come across the error page while navigating our site? Let us know what went
+  wrong
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the error**
+What error message appeared? Was there a status code displayed?
```

Comment:
Can we make this more general `describe the error, the error number if there was one or any additional error context`